Sources/Loaf/Presenter/Controller.swift

```diff
@@ -41,7 +41,9 @@ final class Controller: UIPresentationController {
             containerInsets = UIEdgeInsets(top: min(statusBarSize.width, statusBarSize.height), left: 0, bottom: 0, right: 0)
         }
 
-        if let tabBar = loaf.sender?.parent as? UITabBarController{
+        if let tabBar = loaf.sender?.parent as? UITabBarController {
+            containerInsets.bottom += tabBar.tabBar.frame.height
+        } else if let tabBar = loaf.sender?.parent?.parent as? UITabBarController {
```

Comment:
I wonder if we should also check if the `loaf.sender?.parent` is a `UINavigationController` here, because I worry that the "parent's parent" might be a screen that is hidden from the current view. How does this look?

```suggestion
        } else if let navController = loaf.sender?.parent as? UINavigationController, let tabBar = navController.parent as? UITabBarController {
```